projects/igniteui-angular/src/lib/snackbar/snackbar.component.ts

```diff
@@ -200,6 +220,13 @@ export class IgxSnackbarComponent {
      */
     public snackbarMessage = '';
 
+    /**
+     * @hidden
+     * @internal
+     */
+    public ariaLiveValue = 'polite';
```

Comment:
You don't need a getter/setter, since there's nothing else in the setter, but setting the underlying field value. Remove the getter/setter, make this `ariaLive` field the input and hostbind it and remove it from the template.
```typescript
    /**
     * Sets/gets the `aria-live` attr of the snackbar message container.
     * If not set, the `aria-live` will be `polite`;
     * ```html
     * <igx-snackbar [ariaLive]="'polite'"></igx-snackbar>
     * ```
     * ```typescript
     * let snackbarAriaLive = this.snackbar.ariaLive;
     * ```
     *
     * @memberof IgxSnackbarComponent
     */
    @Input()
    @HostBinding('attr.aria-live')
    public ariaLiveValue = 'polite';
```